polytope/polytope.py

```diff
@@ -400,20 +400,18 @@ def bounding_box(self):
             self.bbox = bounding_box(self)
         return self.bbox
 
-    def plot(self, ax=None, color=None,
-             hatch=None, alpha=1.0):
+    def plot(self, ax=None, color=None, hatch=None, alpha=1.0, linestyle=None, linewidth=None, edgecolor=None):
         if self.dim != 2:
             raise Exception("Cannot plot polytopes of dimension larger than 2")
         ax = _newax(ax)
         if not is_fulldim(self):
-            logger.error("Cannot plot empty polytope")
```

Comment:
I've removed that statement at some point because in my use case it occurred that a polytope to plot was empty a region to plot contained an empty polytope (I don't recall which case occurred). Since the expected behavior for me was that 1) just nothing gets plotted if its empty, and 2) empty polytopes occur, I removed the error warning. I'll restore it for now, if you agree its not the intended behavior I'll remove it again.

Reply:
Thanks for explaining your expectations about behaviour for empty polytopes. I think there are reasons for both (i.e., with and without reporting an error), so let us keep it for now and possibly discuss it more outside this PR.